src/schema_compile.c

```diff
@@ -1486,35 +1486,40 @@ lys_compile_unres_mod_erase(struct lysc_ctx *ctx, ly_bool error)
 }
 
 /**
- * @brief Compile identites in the current module and all its submodules.
+ * @brief Compile identites in a module and all its submodules.
  *
- * @param[in] ctx Compile context.
+ * @param[in] mod Module to process.
  * @return LY_ERR value.
  */
 static LY_ERR
-lys_compile_identities(struct lysc_ctx *ctx)
+lys_compile_identities(struct lys_module *mod)
 {
-    LY_ERR ret = LY_SUCCESS;
+    struct lysc_ctx ctx = {0};
```

Comment:
don't you want to initiate members directly here? Now they are initiated twice actually.

Reply:
Well, it would be possible but I prefer it the way it is now because this initialization simply zeros the memory (like `calloc`) and the later code fills in only the non-NULL members in an orderly fashion.